frontend/src/components/Modal/Modal.styles.ts

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+import { Modal } from 'antd';
+
+export const StyledModal = styled(Modal)`
+  .ant-btn-primary {
+    border-color: ${({ theme }) => theme.colors.primary};
+    background: ${({ theme }) => theme.colors.primary};
+    &:hover {
+      background: ${({ theme }) => theme.colors.hoverPrimary};
+      border-color: ${({ theme }) => theme.colors.hoverPrimary};
+    }
+    &:focus,
+    &:active {
+      background: ${({ theme }) => theme.colors.activePrimary};
+      border-color: ${({ theme }) => theme.colors.activePrimary};
+    }
+  }
+  .ant-btn-default {
+    border-color: ${({ theme }) => theme.colors.primary};
+    color: ${({ theme }) => theme.colors.primary};
+    &:hover {
+      color: ${({ theme }) => theme.colors.hoverPrimary};
+      border-color: ${({ theme }) => theme.colors.hoverPrimary};
+    }
+    &:focus,
+    &:active {
+      border-color: ${({ theme }) => theme.colors.activePrimary};
+      color: ${({ theme }) => theme.colors.activePrimary};
+    }
+  }
+`;
```

Comment:
Czy ten primary button i default nie sa identyczne? Albo podobne? https://github.com/PiotrRynio/CodersCamp2021.Project-4.TaskManager/pull/10#discussion_r864267851 w tym komentarzu tez Piotr mi zrefaktorowal, moze u Ciebie tez sie da pewne powtarzalne linie kodu zebrac razem jak ```.ant-btn-primary {
    border-color: ${({ theme }) => theme.colors.primary}; ``` i ```.ant-btn-default {
    border-color: ${({ theme }) => theme.colors.primary};``` ?

Reply:
Przy korzystaniu z ant design nie mogę użyć tych naszych customowych buttonów, chyba że mogę a nie wiem jak :) dlatego dopasowałam kolory tak żeby się nie kłóciły ze zrobionym już buttonem. Nie są takie same, mają odwrócone kolory, tzn jeden pełny, drugi pusty, zobacze jeszcze czy jakoś mogę to skrócić. Dzięki :)

Reply:
O matko, to kompletnie bez sensu, ze nie da sie uzyc tych customowych :/